package/gluon-core/check_site.lua

```diff
@@ -27,28 +27,31 @@ need_string_array({'ntp_servers'}, false)
 
 need_string_match(in_domain({'prefix6'}), '^[%x:]+/64$')
 
-
+local supported_rates = {6000, 9000, 12000, 18000, 24000, 36000, 48000, 54000}
```

Comment:
This is used to validate mcast_rate entries, I've removed all 802.11b rates here, even when legacy_rates is disabled. Feels a bit weird in that case, but enabling legacy_rates would be equally as weird I suppose.